dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -42,7 +42,9 @@ func checkRootFlags() error {
 // common flags that are used regardless of subcommand type
 // these flags provide additional settings unrelated to tarball generation
 func additionalFlags(cmd *flag.FlagSet) {
-	cmd.StringVar(&customUfsModuleFlag, "custom-ufs-module", "", "a pipe-separated pair of the module name and its comma-separated maven arguments. ex. hadoop-x.y|-pl,underfs/hdfs,-Pufs-hadoop-X,-Dufs.hadoop.version=x.y.z")
+	cmd.StringVar(&customUfsModuleFlag, "custom-ufs-modules", "",
```

Comment:
i'd like to keep the flag name `custom-ufs-module` the same for backcompat, to prevent breaking scripts that call generate-tarball with this flag

Reply:
Thanks for review. The flag name has been fixed back to `custom-ufs-module`